openmetadata-ui/src/main/resources/ui/src/pages/teams/index.test.tsx

```diff
@@ -296,4 +302,19 @@ describe('Test Teams page', () => {
 
     expect(confirmationModal).toBeInTheDocument();
   });
+
+  it('OnClick of manage tab, manage tab content should render', async () => {
+    const { container } = render(<TeamsPage />);
+    const assets = await findByTestId(container, 'manage');
+
+    fireEvent.click(
+      assets,
+      new MouseEvent('click', {
+        bubbles: true,
+        cancelable: true,
+      })
+    );
+
+    expect(await findByText(container, /ManageTab/i)).toBeInTheDocument();
```

Comment:
Can we use test ids and check if the manage tab renders the save button or any other component that we expect to present in the Manage tab after it completes the initial fetch from API?

Reply:
@vivekratnavel , This all things are being covered in manageTab unit test. so here we are just mocking it.